components/comments.jsx

```diff
@@ -12,11 +75,67 @@ const Comments = ({article}) => {
       </div>
       <p className="article-body">{article.body}</p>
       <div className="single-article-line">
-        <p className="article-comment-count">
-          Comment Count: {article.comment_count}
+        <p className="article-comment-count">Comment Count: {commentCount}</p>
+      </div>
+      <button
+        className="btn"
+        onClick={() => {
+          handleClickPostComment();
+        }}
+        id="post-comment-button"
+      >
+        {isPostCommentShown ? "Hide" : "Post a comment"}
+      </button>
+      <div
+        className={`post-comment-${isPostCommentShown ? "shown" : "hidden"}`}
+      >
+        <p className="empty-comment-msg">
+          {isEmpty ? "Please fill in Username & Comment" : ""}
+        </p>
+        <p className="empty-comment-msg">
+          {isSuccessful ? (
+            <></>
+          ) : (
+            "Something has gone wrong, the comment has not been posted."
+          )}
         </p>
+        <div className="single-post-comment-line">
+          <label htmlFor="commenter">Username:</label>
```

Comment:
I would strongly recommend either hard coding a user or using useContext. No recruiter will know which usernames are ok to use here.